build/config.props

```diff
@@ -25,7 +25,7 @@
     <IsEscrowMode Condition="'$(IsEscrowMode)' == ''">false</IsEscrowMode>
 
     <!-- Visual Studio Insertion Logic -->
-    <VsTargetMajorVersion Condition="'$(VsTargetMajorVersion)' == ''">$([MSBuild]::Add(11, $(MajorNuGetVersion)))</VsTargetMajorVersion>
+    <VsTargetMajorVersion Condition="'$(VsTargetMajorVersion)' == ''">$([MSBuild]::Add(12, $(MajorNuGetVersion)))</VsTargetMajorVersion>
```

Comment:
This is actually gonna affect the build numbers as well. 

Basically, InitializeBuild depends on that.